tracer/src/Datadog.Trace/Ci/BenchmarkDiscreteStats.cs

```diff
@@ -0,0 +1,157 @@
+// <copyright file="BenchmarkDiscreteStats.cs" company="Datadog">
+// Unless explicitly stated otherwise all files in this repository are licensed under the Apache 2 License.
+// This product includes software developed at Datadog (https://www.datadoghq.com/). Copyright 2017 Datadog, Inc.
+// </copyright>
+
+#nullable enable
+using System;
+using System.Linq;
+
+namespace Datadog.Trace.Ci;
+
+/// <summary>
+/// Benchmark measurement discrete stats
+/// </summary>
+public readonly struct BenchmarkDiscreteStats
```

Comment:
EDIT: see my next comment first 😅 

Does this _need_ to be public? Could we also make it internal and do an "internals visible to" for the benchmarkdotnet project? 

At the very least we should hide this from being editorbrowseable etc? Same goes for the other types here